src/site/content/en/blog/coop-coep/index.md

```diff
@@ -234,28 +250,35 @@ The parameters object has three properties:
 
 The `group` property names your various reporting endpoints. Use these names to
 direct a subset of your reports. For instance, in the
-`Cross-Origin-Embedder-Policy` directive you can specify the relevant endpoint
-by providing the group name to `report-to=`. For example:
+`Cross-Origin-Embedder-Policy` and `Cross-Origin-Opener-Policy` directives you
+can specify the relevant endpoint by providing the group name to `report-to=`.
+For example:
 
 ```http
-Cross-Origin-Embedder-Policy: require-corp; report-to="coep_rollout_1"
+Cross-Origin-Embedder-Policy: require-corp; report-to="coep_report"
+Cross-Origin-Opener-Policy: same-origin; report-to="coop_report"
 ```
 When the browser encounters this, it will cross reference the `report-to` value
 with the `group` property on the `Report-To` header to look up the endpoint.
-This example cross references on `coep_rollout_1` to find the endpoint
-`https://first-party-test.glitch.me/report`.
+This example cross references on both `coep_report` and `coop_report` to find
+the endpoint `https://first-party-test.glitch.me/report`.
 
-If you prefer to receive reports without blocking any embedded content, use
-`Cross-Origin-Embedder-Policy-Report-Only` instead of
-`Cross-Origin-Embedder-Policy`. For example:
+If you prefer to receive reports without blocking any embedded content or
+isolate a popup window, append `-Report-Only` to respective headers: i.e.
+`Cross-Origin-Embedder-Policy-Report-Only` and
+`Cross-Origin-Opener-Policy-Report-Only`. For example:
 
 ```http
-Cross-Origin-Embedder-Policy-Report-Only: require-corp; report-to="coep_rollout_1"
+Cross-Origin-Embedder-Policy-Report-Only: require-corp; report-to="coep_report"
+Cross-Origin-Opener-Policy-Report-Only: same-origin; report-to="coop_report"
 ```
 
 By doing this, when the browser detects cross origin resources that don't have
 CORP or CORS, it sends a report using the Reporting API without actually
-blocking those resources.
+blocking those resources because of COEP.
+
+Similarly, when the browser opens a cross-origin popup window, it sends a report
+without actually isolating the window because of COOP.
```

Comment:
Note that it also sends reports when two windows that would be placed in different browsing context groups try to access each other. This only works in report-only mode.

Reply:
Thanks Eiji! It looks good.

Reply:
- `This only works in report-only mode` : Is this an interim behavior? 
- `try to access each other`: Any example behaviors?

Reply:
This is not an interim behavior. Making this work in enforcement mode adds a lot of complexity to the code base.

Example:
A page on a.com opens a popup on b.com with COOP report-only same-origin. We get a report for browsing context switch. If the popup does something like window.opener.PostMessage(...), it wouldn't work if we had enforced COOP. With reporting-only mode, we will  get another report that will list the script name and location where the access happened.

Reply:
Got it. Confirmed it emits an error report:

```
[ { age: 51785,
    body:
     { columnNumber: 18,
       disposition: 'reporting',
       effectivePolicy: 'same-origin',
       lineNumber: 83,
       property: 'postMessage',
       sourceFile: 'https://first-party-test.glitch.me/popup.js',
       type: 'access-from-coop-page-to-openee' },
    type: 'coop',
    url:
     'https://first-party-test.glitch.me/coop?report-only&coop=same-origin&',
    user_agent:
     'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4246.0 Safari/537.36' },
  { age: 51785,
    body:
     { disposition: 'reporting',
       effectivePolicy: 'same-origin',
       property: 'postMessage',
       type: 'access-to-coop-page-from-openee' },
    type: 'coop',
    url:
     'https://first-party-test.glitch.me/coop?report-only&coop=same-origin&',
    user_agent:
     'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4246.0 Safari/537.36' } ]
```

Reply:
Address your feedback. Please check again: https://deploy-preview-3797--web-dev-staging.netlify.app/coop-coep/

Reply:
Thanks! Just to note, it's not just for PostMessages though. Any usage of a cross-origin property of the WindowProxies will generate a report.

Reply:
Got approval from Paris. Resolving.